rs/registry/canister/src/registry.rs

```diff
@@ -1358,7 +1372,9 @@ Average length of the values: {} (desired: {})",
             value: original_value.clone(),
         };
         let mut original_registry = Registry::new();
+        let timestamp_before_applying_mutation = Registry::get_current_timestamp_seconds();
         apply_mutations_skip_invariant_checks(&mut original_registry, vec![mutation]);
+        let timestamp_after_applying_mutation = Registry::get_current_timestamp_seconds();
```

Comment:
I think in the ICP, these will always have the same value. IIUC, `time` always returns the same value until you either return from the current canister method call, or call another canister (and neither of those things happens on line 1376).